.travis.yml

```diff
@@ -49,3 +49,13 @@ script:
 
 after_success:
   if [ "$COVER" = "1" ]; then coveralls -v ; fi
+
+deploy:
+  provider: pypi
+  distributions: 'sdist bdist_wheel'
+  user: 'Your username'
+  password:
+    secure: 'Your encrypted password'
+  on:
+    tags: true
+    python: '3.7'
```

Comment:
probably also want to add master branch only

Reply:
It's not possible to use both `tags` and `branch` conditions at the same time.
The `branch` condition will be ignored in this case.

https://docs.travis-ci.com/user/deployment#conditional-releases-with-on
> `tags: true`: deployment is triggered if and only if `$TRAVIS_TAG` is set. Depending on your workflow, you may set `$TRAVIS_TAG` explicitly, even if this is a non-tag build when it was initiated. This causes the `branch` condition to be ignored.